templates/access-rules.yaml.j2

```diff
@@ -0,0 +1,16 @@
+- id: "sample-rule:protected"
```

Comment:
As a freshman in Oathkeeper, I would like to learn:

- For the Oathkeeper's access rules and configuration, do we allow the client user to provide their own access rules and configuration file?
- If this is the case, how do we handle this case? e.g. we try to do a merge of the built-in and user-provided?

Reply:
We will not allow the users to provide their own configuration file (that would be a bit against the idea of operators - charms should encapsulate the necessary logic and configuration), but we will enable admins to provide the access rules.
There will be a relation interface to implement the logic behind it, you can see some context in the Oathkeeper spec. We want to have only one source of the access rules per each app. There will be no built-in rules by default - this one is a sample for testing purposes.

Reply:
ic~~ thank you for the clarifications.